CruCentralCoast/Profile/ChooseCampusOrMovement/ChooseCampusVC.swift

```diff
@@ -0,0 +1,198 @@
+//
+//  ChooseCampusVC.swift
+//  CruCentralCoast
+//
+//  Created by Michael Cantrell on 4/26/18.
+//  Copyright © 2018 Landon Gerrits. All rights reserved.
+//
+
+import UIKit
+import RealmSwift
+
+class ChooseCampusVC: UITableViewController {
+    
+    var subscribedMovements = [String]()
+    var shouldReloadSelectedIndexPath = false
```

Comment:
we try to avoid state variables like this when possible, I'd try to call the reload when you're actually setting this and remove this var altogether

Reply:
However, calling the reload every time a person taps on a Movement in the Choose Movement screen would be more often than needed. The cell only needs to be reloaded once when showing the Choose Campus screen after changing selected movements. The state variable is used to prevent unnecessary reloading of the cell

Reply:
the tech debt with managing state for things that are avoidable will add up, you're right it will be hit more frequently than needed, but I still don't think it'll affect performance, especially because you're just reloading the correct index rather than the whole table which is good. I think robert has just burned into me avoiding state anywhere possible ¯\_(ツ)_/¯ but that's my take

Reply:
I removed the state variable. The code is a little cleaner without it anyway